libraries/GCS_MAVLink/GCS_Param.cpp

```diff
@@ -283,6 +283,26 @@ void GCS_MAVLINK::handle_param_set(const mavlink_message_t &msg)
         return;
     }
 
+    // Check SERVO1_FUNCTION - SERVO16_FUNCTION
+    for (int8_t i = 1; i <= NUM_SERVO_CHANNELS; ++i) {
+        char buf[17];
+        memset(buf, 0, sizeof(buf));
+        hal.util->snprintf(buf, sizeof(buf), "SERVO%d_FUNCTION", i);
+        if (strncmp(key, buf, 16) == 0) {
+            if ((old_value >= 33.0f && old_value <= 40.0f) || (old_value >= 82.0f && old_value <= 85.0f)) {
```

Comment:
These hardcoded values surely are defined as an enum somewhere, aren't they?

Reply:
I still think this whole block should go away personally, it's just a bad approach to the problem.

Reply:
Concur with WickedShell here.  We can't afford flash space for this sort of check.

Any suitably enthusiastic GCS operator could kill the vehicle with the vast majority of our parameters.

Has any vehicle *ever* been lost because the operator decided to change a motor servo function mid-flight?!